src/js/legacy/toggle-veterans-crisis-line.js

```diff
@@ -1,9 +1,17 @@
-$(function() {
-  var toggleVCL = function(e) {
-    e.preventDefault();
-    // Only affects Desktop version of the Veterans Crisis line.
-    // Mobile is handled via menu.js and works with all overlays.
-    $(this).parents('.crisis-line-container').toggleClass('va-crisis-panel--open');
-  }
-  $('a.va-overlay-trigger').on('click', toggleVCL);
+/*
+This script only affects the DESKTOP/WIDESCREEN version
+of the Veteran's Crisis Line trigger
+*/
+document.addEventListener('DOMContentLoaded', () => {
+  const vcls = document.querySelectorAll('.crisis-line-container--wide');
+  const toggleVcl = (vcl) =>
+    vcl.addEventListener('click', (domEvent) => {
+      domEvent.preventDefault();
+      // Check that it was the va-crisis-line trigger element clicked
+      if(domEvent.target.classList.contains('va-crisis-line')) {
+        domEvent.currentTarget.classList.toggle('va-crisis-panel--open');
+      }
+    }); 
+
+  [].map.call(vcls, toggleVcl);
```

Comment:
This is cool.  Am I correct in thinking this calls map on a `NodeList` as if it's an `Array`?